### PR TITLE
Loading indicator for synopsis

### PR DESCRIPTION
**Pull request by K3nnneth**
A loading indicator appears while synopsis is being generated for an article.

Screen Shot 2022-05-15 at 4 25 30 PM

### CODE REVIEW

**Review by EthanEpp — APPROVED** [review]
LGTM!

File: GeoNewsFinder4/components/Overview.js [hunk 1]
```diff
@@ -39,7 +42,10 @@ const OverviewRoute = () => {
   return (
     <View style={styles.overviewContainer}>
       <ScrollView style={styles.scrollView} contentContainerStyle={styles.contentContainer}>
-        <Text style={styles.summaryText}>{summary}</Text>
+        {/* <ActivityIndicator size="large" color="#0000ff" />
```
Comment (mattn02) [inline GeoNewsFinder4/components/Overview.js]: nit: remove comments

File: GeoNewsFinder4/components/Overview.js [hunk 2]
```diff
@@ -39,7 +42,10 @@ const OverviewRoute = () => {
...
+        <Text style={styles.summaryText}>{summary}</Text> */}
```
Comment (mattn02) [inline GeoNewsFinder4/components/Overview.js]: nit: remove comments

**Review by mattn02 — APPROVED** [review]
LGTM just remove commented out components

**Pull request merged**